packages/server-wallet/src/chain-service/chain-service.ts

```diff
@@ -344,27 +344,30 @@ export class ChainService implements ChainServiceInterface {
   private async checkFinalizingChannels(block: providers.Block): Promise<void> {
     const finalizingChannel = this.finalizingChannels.shift();
     if (!finalizingChannel) return;
-
     if (finalizingChannel.finalizesAtS > block.timestamp) {
       this.addFinalizingChannel(finalizingChannel);
       return;
     }
-
-    const {channelId} = finalizingChannel;
-    const [, finalizesAt] = await this.nitroAdjudicator.getChannelStorage(channelId);
-    if (finalizesAt === finalizingChannel.finalizesAtS) {
-      // Should we wait for 6 blocks before emitting the finalized event?
-      // Will the wallet sign new states or deposit into the channel based on this event?
-      // The answer is likely no.
-      // So we probably want to emit this event as soon as possible.
-      this.channelToSubscribers.get(channelId)?.map(subscriber =>
-        subscriber.channelFinalized({
-          channelId,
-        })
-      );
-      // Chain storage has a new finalizesAt timestamp
-    } else if (finalizesAt) {
-      this.addFinalizingChannel({channelId, finalizesAtS: finalizesAt});
+    if (finalizingChannel.finalizesAtS <= block.timestamp) {
```

Comment:
I'm having trouble understanding the reason for this change. This function is not terribly easy to understand in the first place ☹️ .

Reply:
This might be a rebase accident actually, I'll take a look!

Reply:
This has been cleaned up in f6a1027. It should only have been a change to the `args` object but I messed up when rebasing.